.github/workflows/pr-checks.yaml

```diff
@@ -127,7 +127,7 @@ jobs:
           docker info
 
       - name: Acceptance tests
-        timeout-minutes: 60
+        timeout-minutes: 120
```

Comment:
Service and application referenceable both using GVK and GVR introduces changes which would require a change in timeout?